docs/integrations/tax.mdx

```diff
@@ -162,6 +162,12 @@ Avoid synchronous calls to external systems when preparing your tax estimate res
 
 Using [Tax Properties](/docs/store-operations/tax/tax-properties) to enhance the initial Tax Provider API request can help you avoid looking up additional tax estimate inputs on external systems.
 
+#### Infinite loops
+
+Certain BigCommerce API endpoints will automatically invoke an estimate call to a tax provider in certain conditions (e.g. when there is not already a tax quote applied to a cart). These API endpoints must not be called by your tax provider's implementation as doing so will result in an infinite loop of requests, followed by a memory timeout.
```

Comment:
`estimate call to a tax provider under certain conditions` ?

Reply:
rather than `followed by a memory timeout` 
`which will result in a memory timeout`

Reply:
More specific/less fluffy - lovely. Thanks!

Reply:
Adjusted a couple of items in this area, cheers.

Realised I had a double "Certain" also 😆